src-tauri/src/gpu_miner_adapter.rs

```diff
@@ -218,6 +229,27 @@ impl StatusMonitor for GpuMinerStatusMonitor {
 
     async fn status(&self) -> Result<Self::Status, anyhow::Error> {
         let client = reqwest::Client::new();
+        let config_path = PathBuf::from(""); //TODO add app config_path
```

Comment:
if you move the config path in the gpuminer to after the config file, you will not have this problem

Reply:
Thanks! I'll refactor this after merge this and 612 - I'll have to add some code to integrate these two PRs anyway